python/bigipconfigdriver.py

```diff
@@ -375,14 +376,22 @@ def create_ltm_config_kubernetes(bigip, config):
                 f5_service['virtual_address'] = \
                     svc['virtualAddress']['bindAddr']
 
+                addr = svc['virtualAddress']['bindAddr']
+                port = svc['virtualAddress']['port']
+                destination = None
+                if isinstance(ipaddress.ip_address(addr),
```

Comment:
The controller's schema will ensure this is a valid address, correct? I think we can rely on that but otherwise should probably catch ipaddress.AddressValueError.